packages/react-components/react-radio/src/components/RadioGroup/RadioGroup.types.ts

```diff
@@ -63,6 +68,17 @@ export type RadioGroupOnChangeData = {
   value: string;
 };
 
+/**
+ * Imperative API for RadioGroup.
+ */
+export type RadioGroupImperativeRef = {
+  /**
+   * Calls focus() on the selected radio item inside this RadioGroup.
+   * If none are selected, calls focus() on the first radio item.
+   */
+  focusSelected: () => void;
```

Comment:
Should this be called `focus()` rather than `focusSelected()`? My reasoning:

1. `focus()` is the standard browser API for focusing an element so any users would likely expect to call that rather then `focusSelected()`
2. This is an API on `RadioGroup` which is a `div` and not focusable by default so overriding `focus()` in this case is safe and "does the right thing"